### PR TITLE
mongodb 3.1.1

### PR DESCRIPTION
**Pull request by ehershey**

### CODE REVIEW

**Comment by mikemcquaid** [comment]
devel needs a `fails_with` or `needs :cxx11` it seems: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/23918/version=mountain_lion/testReport/junit/brew-test-bot/mountain_lion/install___devel_mongodb/

**Comment by tdsmith** [comment]
Or just `ENV.libcxx` in `install`, maybe?

**Comment by ehershey** [comment]
I added ENV.libcxx. It seems like that should work. If it doesn't work I can play around locally in 10.8.

File: Library/Formula/mongodb.rb [hunk 1]
```diff
@@ -30,6 +40,10 @@ class Mongodb < Formula
 
   def install
 
+    # The libstdc++ on old OS versions is too old so use libcpp instead
+    #
```
Comment (mikemcquaid) [inline Library/Formula/mongodb.rb]: You can kill the comment here and newline above it.

File: Library/Formula/mongodb.rb [hunk 2]
```diff
@@ -30,6 +40,10 @@ class Mongodb < Formula
...
+    ENV.libcxx
```
Comment (mikemcquaid) [inline Library/Formula/mongodb.rb]: `if build.devel?`

**Comment by tdsmith** [comment]
I pulled 3.0.2 stable so this will need to be rebased on master.

**Comment by ehershey** [comment]
Thanks! Sorry I've been slow on getting this updated.

**Comment by ehershey** [comment]
I rebased, took out the comment, and added "if build.devel?" for ENV.libcxx. I tested a bunch too so hopefully this is good to go.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple devel updates is `mongodb 3.1.1 (devel)`.

**Pull request closed without merging**